apps/widget-configurator/src/app/configurator/consts.ts

```diff
@@ -94,6 +94,11 @@ export const COW_LISTENERS: CowEventListeners = [
     event: CowEvents.ON_FULFILLED_ORDER,
     handler: (event) => console.log(`[configurator:ON_FULFILLED_ORDER] ✅ Executed order ${event.order.uid}`),
   },
+
+  {
+    event: CowEvents.ON_TRADE_PARAMS,
```

Comment:
Maybe a verb is missing? 

`ON_FULFILLED_ORDER`  here an order was fulfilled. 

`TRADE_PARAMS` is just a noun. Maybe `ON_CHANGE_TRADE_PARAMS`

Reply:
Thanks! Renamed the const